demo/molecule/dataCounter/demo/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "demo",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@adv-ui/fc-iconset": "3",
+    "@s-ui/react-molecule-data-counter": "1"
```

Comment:
This package might be imported from components directory (and it is), not the published. If its not you cant show demo features until it's already merged. This dependency is not necesary